common/setups/rasr/util.py

```diff
@@ -8,8 +8,13 @@
     "GmmSatArgs",
     "GmmVtlnSatArgs",
     "ForcedAlignmentArgs",
+    "RecognitionArgs",
```

Comment:
Do you think it would make sense to separate this further? I imagine that GmmRecogntionArgs, HybridRecognitionArgs, CTCRegognitionArgs ect.... might be difficult.

Reply:
I am thinking that maybe we could merge the `Gmm*Args` to `GmmStepArgs` because basically it is always the same steps and then make an documentation at the beginning of the file.

at the moment RecognitionArgs only holds the name plus a dict with the arguments.. I dont see a need to split this..

I generally agree that the `util.py` is a bit big

Reply:
split maybe into `gmm_util`, `nn_util`, `ctc_util` or so?